build-time-analytics/src/test/java/io/quarkus/ts/buildtimeanalytics/AbstractAnalyticsIT.java

```diff
@@ -143,7 +143,7 @@ private void validateExtensions(QuarkusCliRestService app, JsonPath json) {
         assertEquals(pomDependencyGAs, payloadExtensionGAs);
 
         // RHBQ doesn't guarantee the same version of the platform and core extensions
-        boolean isRHBQ = QuarkusProperties.getVersion().contains("redhat");
+        boolean isRHBQ = QuarkusProperties.isRHBQ();
```

Comment:
Looking at this and think this can be moved to if as it not used anywhere else and it's only condition

Reply:
Fixed it